src/Components/NavBar/NavBar.tsx

```diff
@@ -240,7 +245,7 @@ const MobileMenu = (props: Props) => {
         <FeatherIcon icon="menu" />
       </MobileFeatherIconWrapper>
       <Portal>
-        <StyledPortalWrapper animate={controls}>
+        <StyledPortalWrapper height={finalHeight} animate={controls}>
```

Comment:
Maybe `finalHeight` can have other name more accurate to the wrapper

Reply:
Gotcha